### PR TITLE
Platform - UI: Minor UI updates

### PR DESCRIPTION
**Pull request by deetz99**
*Description of changes:*
- fix phone input not unmasking value
- missing i18n translations for 'certify' and confirmation checkbox error messages
- add nameLocal as hidden text for screen readers to country code dropdown
- move button control buttons to right side per Scotts request

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the strr license (Apache 2.0).

### CODE REVIEW

File: strr-platform-web/app/components/connect/ButtonControl.vue
Comment (deetz99) [inline strr-platform-web/app/components/connect/ButtonControl.vue:1]: Had to remove col-auto and use col-start-* as empty columns don't have a width so the right buttons wouldn't be at the right side of the container

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://strr-platform-dev--pr-233-6lll0f9l.web.app

File: strr-platform-web/app/components/connect/ButtonControl.vue
Comment (deetz99) [reply strr-platform-web/app/components/connect/ButtonControl.vue:1]: Updates name from +1 -> +1 Canada

Screenshot 2024-10-23 at 5 28 22 PM

File: strr-platform-web/app/components/connect/form/phoneNumber/CountryCode.vue
Comment (deetz99) [inline strr-platform-web/app/components/connect/form/phoneNumber/CountryCode.vue:1]: Updates name from +1 -> +1 Canada

Screenshot 2024-10-28 at 10 23 16 AM

File: strr-platform-web/app/stores/platformApplication.ts
Comment (deetz99) [inline strr-platform-web/app/stores/platformApplication.ts:1]: Custom validation messages weren't working with z.literal so switched to refine

**Review by kialj876 — APPROVED** [review]

**Pull request merged**